source/guides/get-help-from-other-roles.html.md

```diff
@@ -42,3 +42,38 @@ To get help
 - send a direct message on Slack
 - use a collaborative tool such as Padlet, Trello, Mural or Google Doc to frame your thoughts
 - arrange a call  
+
+## Designers on the Design System team 
+The Design System team has a number of different designers covering the interaction, service and graphic design disciplines. They use their knowledge of user needs to design styles, components and patterns that assist the community. 
+
+As a group of designers on this team, we tend not to focus on job titles and specific DDAT skills. Instead we work as a group. Though each of us will have particular areas of expertise, we will often work together to draw from each other's skills and knowledge.
+
+### Designers on the team
+Ciandelle - I have a background in both service and interaction design. I prefer to work on things where I can pay close attention to detail and visualise the work as part of the larger picture. I am really confident with the prototype kit. I struggle with physical drawing and graphic design. 
+
+Chris - I’m part of the team’s leadership group, so I’m involved in shaping the team’s priorities and improving how we work. If you’re not sure about how to involve design in something you’re working on, come to me! I also spend plenty of my time doing more hands-on design work, and am comfortable working across service, interaction and graphic design. I have also been at GDS and on the Design System a pretty long time, so get in touch if you’re after historical context on something – if I don’t know, I should know who does know.
+
+Charlotte - I trained as a product designer with a bit of engineering but always preferred service design projects. GDS hired me as a motion graphic designer and illustrator, I mainly did slide decks, animations and social media assets. I’m happy to lend my hand to anything, I like to look at things in detail using grids and maths (engineering side) but I struggle with being super creative. I’m more iterative than innovative. 
+
+Mia - I trained as a graphic designer but since then I’ve done a range of things, including motion graphics, product design, interaction design, illustration, publication design and exhibition design. I can draw you a logo and I can help you refine your brand. Coding in HTML and CSS are fun things I like to do when the need arises, but I’m not up on the latest frameworks. I’m generally tool agnostic, and I will always start from first principles: why does this exist, what are we trying to do, what are the words we’re using? My philosophy is: don’t try to solve a content problem with interaction design. Also, if it’s not accessible, it’s not finished.
```

Comment:
```suggestion
#### Mia
I trained as a graphic designer but since then I’ve done a range of things, including motion graphics, product design, interaction design, illustration, publication design and exhibition design. I can draw you a logo and I can help you refine your brand. Coding in HTML and CSS are fun things I like to do when the need arises, but I’m not up on the latest frameworks. I’m generally tool agnostic, and I will always start from first principles: why does this exist, what are we trying to do, what are the words we’re using? My philosophy is: don’t try to solve a content problem with interaction design. Also, if it’s not accessible, it’s not finished.
```